### PR TITLE
jenkins: use lower segment size for back comp test

### PR DESCRIPTION
**Pull request by egonelbre**
What: Currently backwards compatibility test uses a lot of resources due to needing to upload 128MB to 10 storage nodes. Reduce this to 13MB and set segment size to 6MB.

Why:

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: scripts/test-backwards.sh [hunk 1]
```diff
@@ -15,9 +15,9 @@ setup(){
         output=$2
 	    head -c $size </dev/urandom > $output
     }
-    random_bytes_file "2K"   "$TEST_FILES_DIR/small-upload-testfile"          # create 2kb file of random bytes (inline)
-    random_bytes_file "5M"   "$TEST_FILES_DIR/big-upload-testfile"            # create 5mb file of random bytes (remote)
-    random_bytes_file "128M" "$TEST_FILES_DIR/multisegment-upload-testfile"   # create 128mb file of random bytes (remote)
+    random_bytes_file "2K"   "$TEST_FILES_DIR/small-upload-testfile"        # create 2kb file of random bytes (inline)
+    random_bytes_file "5M"   "$TEST_FILES_DIR/big-upload-testfile"          # create 5mb file of random bytes (remote)
+    random_bytes_file "13M" "$TEST_FILES_DIR/multisegment-upload-testfile"  # create 13mb file of random bytes (remote)
```
Comment (littleskunk) [inline scripts/test-backwards.sh]: This file is special. It needs to be exactly 2 segments. Without the reduced segments size a 129M will work but the 128M file triggered an error doing the last release test. 129M is 3 segments. 128M is 2 segments + 1 inline segment. We need to cover that inline segment.

File: scripts/test-sim-backwards.sh [hunk 2]
```diff
@@ -47,6 +47,9 @@ PATH=$RELEASE_DIR/bin:$PATH storj-sim -x --host $STORJ_NETWORK_HOST4 network --p
 # run upload part of backward compatibility tests from the lastest release branch
 PATH=$RELEASE_DIR/bin:$PATH storj-sim -x --host $STORJ_NETWORK_HOST4 network test bash "$SCRIPTDIR"/test-backwards.sh upload
 
+# set the segment size lower to make test run faster
+echo client.segment-size: 6.0 MiB >> `storj-sim network env GATEWAY_0_DIR`/config.yaml
```
Comment (littleskunk) [inline scripts/test-sim-backwards.sh]: gateway config should have no affect. We are testing uplink.

Comment (egonelbre) [reply scripts/test-sim-backwards.sh]: Uplink is using gateway config, because we haven't added automatic uplink config creation yet.

**Comment by mniewrzal** [comment]
@egonelbre I was able to trigger error that is covered by 64MiB segment size with `--client.segment-size` on bucket.

**Comment by egonelbre** [comment]
@mniewrzal thanks :)

**Pull request merged**